### PR TITLE
Prop Possesion: Made input directions more intuitive

### PR DESCRIPTION
**Pull request by TimGoll**
Reworked the propspec inputs to be more predictable. The old implementation was really janky and did not take into account in which way the player is looking, see #1093 

I also removed the angular force, because I'm pretty sure it was only added to "fix" the broken implementation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 - Fixed blurred box rendering for boxes not starting at `0,0` (by @TimGoll)
 - Optimized allocations by using global Vector / Angle when possible.
 - Fixed the dynamic armor damage calculation being wrong when damage can only get partially reduced
+- Fixed propspec inputs being sometimes unpredictable (by @TimGoll)
```
Comment (saibotk) [inline CHANGELOG.md:50]: Can we use another terminology, as prediction here is usually referred to networking.

**Review by saibotk — CHANGES_REQUESTED** [review]

**Review by saibotk — APPROVED** [review]
Very nice!

File: gamemodes/terrortown/gamemode/server/sv_propspec.lua [hunk 2]
```diff
@@ -147,42 +147,44 @@ function PROPSPEC.Key(ply, key)
...
 
-		pr.t = CurTime() + 0.05
+		pSpec.t = CurTime() + 0.05
 	elseif key == IN_FORWARD then
-		-- bump away from player
-		phys:ApplyForceCenter(aim * mf)
+		phys:ApplyForceCenter(vectorAimPlanar * mf)
 	elseif key == IN_BACK then
-		phys:ApplyForceCenter(aim * (mf * -1))
+		phys:ApplyForceCenter(vectorAimPlanar * (mf * -1))
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/server/sv_propspec.lua:176]:
```suggestion
		phys:ApplyForceCenter(-vectorAimPlanar * mf)
```

**Review by Alf21 — APPROVED** [review]
Nice!

**Pull request merged**